nautobot/docs/development/core/index.md

```diff
@@ -359,3 +359,7 @@ The change summary is added to the file in plain text. Change summaries should b
     ```plaintext title="changes/1234.changed"
     Changed release notes generation.
     ```
+
+### Documentation Updates
+
+The main landing page needs to remain at `docs/index.md` to keep the landing page documentation going to the appropriate location.
```

Comment:
I wasn't sure what this meant until I read the issue and understood the problem you're fixing here. Perhaps changing to something like this would be clearer?
```suggestion
The main landing page needs to remain at `docs/index.md` to prevent 404 errors when switching documentation versions on readthedocs.
```